enhancements/monitoring/user-workload-monitoring.md

```diff
@@ -136,6 +136,26 @@ As a service owner, I'd like to configure alerting rules or recording rules for
 
 As a service owner, I'd like to view configured and fired alerts in the dev perspective of OpenShift console.
 
+### US11
+
+As a member of the operations team, I’d like to get notified if the Prometheus servers are acting strange.
+
+### US12
+
+As a member of the operations team, I'd like to have an overview about all the Prometheus servers inside a cluster.
+
+### US13
+
+As a member of the operations team, I'd like to configure more replicas for the new Prometheus servers.
```

Comment:
And rulers?

Reply:
ah, good catch, we can add this indeed, potentially as post-GA, wdyt?